APCAppCore/APCAppCore/Library/Categories/NSError+APCAdditions.m

```diff
@@ -31,23 +31,19 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. 
 // 
  
-#import "APCAppCore.h"
-
-static NSString*    kServerBusy                     = @"Thank you for your interest in this study. We are working hard to process the large volume of interest, and should be back up momentarily. Please try again soon.";
-static NSString*    kUnexpectConditionMessage       = @"An unexpected network condition has occurred. Please try again soon.";
-static NSString*    kNotConnectedMessage            = @"You are currently not connected to the Internet. Please try again when you are connected to a network.";
-static NSString*    kServerMaintanenceMessage       = @"The study server is currently undergoing maintanence. Please try again soon.";
-static NSString*    kAccountAlreadyExists           = @"An account has already been created for this email address. Please use a different email address, or sign in using the \"already participating\" link at the bottom of the Welcome page.";
-static NSString*    kAccountDoesNotExists           = @"There is no account registered for this email address.";
-static NSString*    kBadEmailAddress                = @"The email address submitted is not a valid email address. Please correct the email address and try again.";
-static NSString*    kBadPasswordAddress             = @"The password you have entered is not a valid password.  Please try again.";
-static NSString*    kNotReachableMessage            = @"We are currently not able to reach the study server. Please retry in a few moments.";
-static NSString*    kInvalidEmailAddressOrPassword  = @"Entered email address or password is not valid. Please correct the email address or password and try again.";
-static NSString*    kSageMessageKey                 = @"message";
-static NSString*    kSageErrorsKey                  = @"errors";
-static NSString*    kSageErrorPasswordKey           = @"password";
-static NSString*    kSageErrorEmailKey              = @"email";
-static NSString*    kSageInvalidUsernameOrPassword  = @"Invalid username or password.";
+#import "NSError+APCAdditions.h"
+#import "APCLog.h"
+
+NSString * const kServerBusy                     = @"Thank you for your interest in this study. We are working hard to process the large volume of interest, and should be back up momentarily. Please try again soon.";
```

Comment:
These constants should be prefixed with "APC" if there are to have external linkage.